business_management/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Bussiness  Management",
+    "version": "15.0.0.0.1",
+    "category": "Bussiness management",
+    "summary": "Bussiness Management for All Bussiness",
+    "website": "https://github.com/OCA/product-variant",
+    "depends": [
+        "mrp",
+        "sale_project",
+        "purchase",
+    ],
+    "data": [
+        "views/sale_order_views.xml",
+        "views/account_move_views.xml",
+        "views/stock_picking_views.xml",
+        "views/purchase_order_views.xml",
+        "views/mrp_production_viwes.xml",
+        "views/project_views.xml",
+        "views/project_task_views.xml",
+    ],
+    "installable": True,
+    "application": True,
```

Comment:
Business flow is module not application, please remove application=True from the manifest